source/CMakeLists.txt

```diff
@@ -8,14 +8,16 @@ add_subdirectory(utils)
 
 # HDF5 VOL requires 1.13+
 if(ADIOS2_HAVE_HDF5)
-  if(HDF5_VERSION VERSION_LESS 1.13)
+  if(HDF5_VERSION VERSION_LESS 1.14)
```

Comment:
This version conditional should be done in `cmake/DetectOptions.cmake:308`

Reply:
Oh, as discussed yesterday in the call, our VOL support will require the most  recent hdf5 version. 

Other HDF5 related features can work with older versions. 

That's why I put a version check here.

Reply:
ohh that is right